### DISCUSSION TITLE
error:0A000152:SSL routines::unsafe legacy renegotiation disabled

### DISCUSSION THREAD

**Discussion by Wolke24** [idaholab/moose#23826] [Q&A Getting Started] [answered]
## Bug Description
(base) ➜  ~ mamba create -n moose moose-tools moose-libmesh

                  __    __    __    __
                 /  \  /  \  /  \  /  \
                /    \/    \/    \/    \
███████████████/  /██/  /██/  /██/  /████████████████████████
              /  / \   / \   / \   / \  \____
             /  /   \_/   \_/   \_/   \    o \__,
            / _/                       \_____/  `
            |/
        ███╗   ███╗ █████╗ ███╗   ███╗██████╗  █████╗
        ████╗ ████║██╔══██╗████╗ ████║██╔══██╗██╔══██╗
        ██╔████╔██║███████║██╔████╔██║██████╔╝███████║
        ██║╚██╔╝██║██╔══██║██║╚██╔╝██║██╔══██╗██╔══██║
        ██║ ╚═╝ ██║██║  ██║██║ ╚═╝ ██║██████╔╝██║  ██║
        ╚═╝     ╚═╝╚═╝  ╚═╝╚═╝     ╚═╝╚═════╝ ╚═╝  ╚═╝

        mamba (1.0.0) supported by @QuantStack

        GitHub:  https://github.com/mamba-org/mamba
        Twitter: https://twitter.com/QuantStack

█████████████████████████████████████████████████████████████

WARNING: A conda environment already exists at '/Users/chuxiaoyun/mambaforge3/envs/moose'
Remove existing environment (y/[n])? y

Looking for: ['moose-tools', 'moose-libmesh']

conda-forge/osx-arm64                               2.2s
conda-forge/noarch                                  3.0s
Download error (35) SSL connect error [https://conda.software.inl.gov/public/noarch/repodata.json]
error:0A000152:SSL routines::unsafe legacy renegotiation disabled

# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "/Users/chuxiaoyun/mambaforge3/lib/python3.10/site-packages/conda/exceptions.py", line 1129, in __call__
        return func(*args, **kwargs)
      File "/Users/chuxiaoyun/mambaforge3/lib/python3.10/site-packages/mamba/mamba.py", line 935, in exception_converter
        raise e
      File "/Users/chuxiaoyun/mambaforge3/lib/python3.10/site-packages/mamba/mamba.py", line 928, in exception_converter
        exit_code = _wrapped_main(*args, **kwargs)
      File "/Users/chuxiaoyun/mambaforge3/lib/python3.10/site-packages/mamba/mamba.py", line 886, in _wrapped_main
        result = do_call(parsed_args, p)
      File "/Users/chuxiaoyun/mambaforge3/lib/python3.10/site-packages/mamba/mamba.py", line 754, in do_call
        exit_code = create(args, parser)
      File "/Users/chuxiaoyun/mambaforge3/lib/python3.10/site-packages/mamba/mamba.py", line 630, in create
        return install(args, parser, "create")
      File "/Users/chuxiaoyun/mambaforge3/lib/python3.10/site-packages/mamba/mamba.py", line 497, in install
        index = load_channels(pool, channels, repos)
      File "/Users/chuxiaoyun/mambaforge3/lib/python3.10/site-packages/mamba/utils.py", line 129, in load_channels
        index = get_index(
      File "/Users/chuxiaoyun/mambaforge3/lib/python3.10/site-packages/mamba/utils.py", line 110, in get_index
        is_downloaded = dlist.download(api.MAMBA_DOWNLOAD_FAILFAST)
    RuntimeError: Download error (35) SSL connect error [https://conda.software.inl.gov/public/noarch/repodata.json]
    error:0A000152:SSL routines::unsafe legacy renegotiation disabled

`$ /Users/chuxiaoyun/mambaforge3/bin/mamba create -n moose moose-tools moose-libmesh`

  environment variables:
                 CIO_TEST=<not set>
        CONDA_DEFAULT_ENV=base
                CONDA_EXE=/Users/chuxiaoyun/mambaforge3/bin/conda
             CONDA_PREFIX=/Users/chuxiaoyun/mambaforge3
    CONDA_PROMPT_MODIFIER=(base)
         CONDA_PYTHON_EXE=/Users/chuxiaoyun/mambaforge3/bin/python
               CONDA_ROOT=/Users/chuxiaoyun/mambaforge3
              CONDA_SHLVL=1
           CURL_CA_BUNDLE=<not set>
                 INFOPATH=/opt/homebrew/share/info:
                  MANPATH=/opt/local/share/man:/opt/homebrew/share/man::
                     PATH=/Users/chuxiaoyun/mambaforge3/bin:/Users/chuxiaoyun/mambaforge3/condab
                          in:/opt/local/bin:/opt/local/sbin:/opt/homebrew/bin:/opt/homebrew/sbin
                          :/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/
                          Library/Frameworks/Mono.framework/Versions/Current/Commands
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>

     active environment : base
    active env location : /Users/chuxiaoyun/mambaforge3
            shell level : 1
       user config file : /Users/chuxiaoyun/.condarc
 populated config files : /Users/chuxiaoyun/mambaforge3/.condarc
                          /Users/chuxiaoyun/.condarc
          conda version : 22.9.0
    conda-build version : not installed
         python version : 3.10.6.final.0
       virtual packages : __osx=11.6.5=0
                          __unix=0=0
                          __archspec=1=arm64
       base environment : /Users/chuxiaoyun/mambaforge3  (writable)
      conda av data dir : /Users/chuxiaoyun/mambaforge3/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.software.inl.gov/public/osx-arm64
                          https://conda.software.inl.gov/public/noarch
                          https://conda.anaconda.org/conda-forge/osx-arm64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /Users/chuxiaoyun/mambaforge3/pkgs
                          /Users/chuxiaoyun/.conda/pkgs
       envs directories : /Users/chuxiaoyun/mambaforge3/envs
                          /Users/chuxiaoyun/.conda/envs
               platform : osx-arm64
             user-agent : conda/22.9.0 requests/2.28.1 CPython/3.10.6 Darwin/20.6.0 OSX/11.6.5
                UID:GID : 501:20
             netrc file : None
           offline mode : False

An unexpected error has occurred. Conda has prepared the above report.

**Comment by geoyanzhan3** [comment]
I got the same error, when I install MOOSE on a new machine.

**Comment by aiskhak** [comment]
Same issue

**Comment by GiudGiud** [comment, marked as answer]
Hello

Sorry we dont handle user support under Issues but under Discussions. 
I think the issue has been resolved with time. Please let us know if you still hit it.

This is likely a conda error anyway, and you can do
`SSL_NO_VERIFY=1 mamba update`

Guillaume
